### PR TITLE
Rebuild for Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #11 
Fixes #12

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,23 +36,24 @@ source:
     - univariate_test.py.patch
 
 build:
-  number: 0
+  number: 1001
   skip: true      # [win or py3k or np>111]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Think we can drop the `np` part of this selector.

**Comment by jakirkham** [comment]
Thanks for working on this, @xhochy. One small comment above.

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Comment by jakirkham** [comment]
Thanks @xhochy 😄

**Comment by jakirkham** [comment]
@conda-forge-admin, please update circle.

**Pull request merged**